.github/workflows/checks-audit.yaml

```diff
@@ -19,7 +19,7 @@ concurrency:
 jobs:
   cargo-audit:
     name: Audit
-    runs-on: self-hosted-hoprnet-big
+    runs-on: self-hosted-hoprnet-bigger
```

Comment:
This could be a small runner imho. Just a fast check of deps.

Reply:
Yes, but the issue is that we need to get rid of the big. Because they don't feet in the Kubernetes